specification/2.0/Specification.adoc

```diff
@@ -1340,25 +1361,48 @@ A significant number of authoring and client implementations associate names wit
+The order of joints is defined by the `skin.joints` array and it **MUST** match the order of `inverseBindMatrices` accessor elements (when the latter is present). The `skeleton` property (if present) points to the node that is the common root of a joints hierarchy or to a direct or indirect parent node of the common root.
 
 [NOTE]
 .Implementation Note
 ====
-Although the `skeleton` property is not needed for computing skinning transforms, it may be used to provide a specific "pivot point" for the skinned geometry.
+Although the `skeleton` property is not needed for computing skinning transforms, it may be used to provide a specific "`pivot point`" for the skinned geometry.
 ====
 
 An accessor referenced by `inverseBindMatrices` **MUST** have floating-point components of `"MAT4"` type. The number of elements of the accessor referenced by `inverseBindMatrices` **MUST** greater than or equal to the number of `joints` elements.
 
 [NOTE]
 .Implementation Note
 ====
-The matrix defining how to pose the skin's geometry for use with the joints (also known as "Bind Shape Matrix") should be premultiplied to mesh data or to Inverse Bind Matrices.
+The matrix defining how to pose the skin's geometry for use with the joints (also known as "`Bind Shape Matrix`") should be premultiplied to mesh data or to Inverse Bind Matrices.
 ====
 
-Only the transform of the skeleton root node is applied to the skinned mesh while the transform of the skinned mesh node **MUST** be ignored.
+[[joint-hierarchy]]
+==== Joint Hierarchy
+
+The joint hierarchy used for controlling skinned mesh pose is simply the node hierarchy, with each node designated as a _joint_ by a reference from the `skin.joints` array. Each skin's joints **MUST** have a common parent node (direct or indirect) called _common root_, which may or may not be a joint node itself. When a skin is referenced by a node within a scene, the common root **MUST** belong to the same scene.
+
+[NOTE]
+.Implementation Note
+====
+A node object does not specify whether it is a joint. Client implementations may need to traverse the `skins` array first, marking each joint node.
+====
+
+A joint node **MAY** have regular nodes attached to it, even a complete node sub graph with meshes.
+
+[NOTE]
+.Implementation Note
+====
+It's often used to have an entire geometry attached to a joint node without having it being skinned (e.g., a sword attached to a hand). Note that the node transform is the local transform of the node relative to the joint, like any other node in the glTF node hierarchy as described in the <<transformations,Transformations>> section.
+====
+
+To apply skinning, a regular transformation matrix is computed for each joint and then used to transform individual vertices as described in the next section.
```

Comment:
I'm not sure I understand the purpose of this statement. What is the ideal behind adding this?

Reply:
The older version of the spec didn't mention what to do with the joint matrices at all; it just had a link to the overview image that is outdated now.

Reply:
Ok, does this line need to be here if it's used only in the next section?